File: 백준/Bronze/1546. 평균/평균.c
int main()
{
    int N;
    scanf("%d\n", &N);

    int score[N];
    int max = 0;

    for(int i=0; i<N; i++){
        scanf("%d", &score[i]);

        if(score[i] > max)
            max = score[i];
    }

    double sum = 0; //왜 double을 써야 하지 ?
    for(int i=0; i<N; i++){
        sum += ((double) score[i]/max)*100; // double!
    }

    printf("%f", (double)sum/N);

    return 0;
}

File: 백준/Bronze/1546. 평균/평균.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().split("\n");

let n = Number(input[0]);
let arr = input[1].split(" ").map(Number);

let max = 0;
for (let i = 0; i < n; i++) {
  max = Math.max(arr[i], max);
}

let sum = 0;
for (let i = 0; i < n; i++) {
  sum += arr[i] / max * 100;
}

console.log(sum / n);